javascript/chronometer.js

```diff
@@ -1,26 +1,41 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    const incrementSec = setInterval(() => {
+      this.currentTime++; 
+    }, 1000);
```

Comment:
Estás redeclarando una variable incrementSec, el objetivo es parar el setInterval que se ha iniciado con startClick() y has guardado en this.intervalId, entonces:
`return clearInterval(this.intervalId);`